pagopa/pn-frontend
# feat(PN-7010) - show analog flow events representing courier errors

## Short description
Now the SEND_ANALOG_PROGRESS events having either CON993 / CON995 / CON996 / CON997 / CON998 as value for `deliveryDetailCode`, are considered in the rendering of a notification timeline for PA / PF / PG.

## List of changes proposed in this pull request
- Added the codes into the list of allowed analog flow codes to be shown in timeline.
- Provided title and descriptions, as specified in https://pagopa.atlassian.net/wiki/spaces/PN/pages/629014723/Timeline+events+e+cosa+mostrare+agli+utenti#Tabella-di-approfondimento-SEND_ANALOG.

## How to test
Enter the details for a notification including the kind of events which are the subject of this PR. The events should be included in timeline with the expected title and description.  
Current examples in TEST
- IUN QYAN-NKHR-QDEX-202307-K-1- comune di Sappada / leonardo - includes a event with `deliveryDetailCode` CON996 .
- IUN REGE-JDZD-TWUM-202307-E-1- comune di Sappada / leonardo - includes a event with `deliveryDetailCode` CON997 .
 
You can create more examples in DEV or TEST using these special postal addresses
- Via@FAIL_CONSOLIDATORE-AR - to include a CON996 event.
- Via@FAIL_CONSOLIDATOREINDIRIZZO-AR - to include a CON997 event.

Unfortunately no procedures have been provided for CON993 / CON995 / CON998, which have the same title and description as CON996. Of course, you can mock the `getSentNotification` / `getReceivedNotification` in the Notification API in order to change e.g. CON996 for any of its "siblings".

**Important note**
These events halt the evolution of a notification, i.e. no further events are expected after a CON99x. _This is the expected behavior_, don't take it as an error.
